workflows/phylogenetics/wf_snippy_streamline_fasta.wdl

```diff
@@ -0,0 +1,109 @@
+version 1.0
+
+import "../../tasks/phylogenetic_inference/utilities/task_centroid.wdl" as centroid_task
+import "../../tasks/phylogenetic_inference/utilities/task_referenceseeker.wdl" as referenceseeker_task
+import "../../tasks/task_versioning.wdl" as versioning
+import "../../tasks/utilities/data_import/task_ncbi_datasets.wdl" as ncbi_datasets_task
+import "../../workflows/phylogenetics/wf_snippy_tree.wdl" as snippy_tree_workflow
+import "../../workflows/standalone_modules/wf_snippy_variants.wdl" as snippy_variants_workflow
+
+workflow snippy_streamline_fasta {
+  input {
+    Array[File] assembly_fasta
+    Array[String] samplenames
+    String tree_name
+    String tree_name_updated = sub(tree_name, " ", "_")
```

Comment:
Can you move this out of the input section to the line beneath it so it doesn't show up on Terra?